x-pack/plugins/siem/public/timelines/components/timeline/properties/helpers.tsx

```diff
@@ -160,28 +176,45 @@ export const NewCase = React.memo<NewCaseProps>(
 );
 NewCase.displayName = 'NewCase';
 
-interface NewTimelineProps {
+export interface NewTimelineProps {
   createTimeline: CreateTimeline;
   onClosePopover: () => void;
+  outline?: boolean;
+  showTimeline?: (args: { id: string; show: boolean }) => void;
   timelineId: string;
+  timelineType: TimelineTypeLiteral;
+  title?: string;
 }
 
 export const NewTimeline = React.memo<NewTimelineProps>(
-  ({ createTimeline, onClosePopover, timelineId }) => {
+  ({
+    createTimeline,
+    onClosePopover,
+    outline = false,
+    showTimeline = noop,
+    timelineId,
+    timelineType = TimelineType.default,
+    title,
+  }) => {
     const handleClick = useCallback(() => {
-      createTimeline({ id: timelineId, show: true });
+      createTimeline({ id: timelineId, show: true, timelineType });
       onClosePopover();
+      showTimeline({ id: timelineId, show: true });
```

Comment:
why do we need to call `showTimeline`? I think `createTimeline` with`show: true` should be enough?